### PR TITLE
Third task protosevich

### PR DESCRIPTION
**Pull request by kissipat**
Hi, check please my third homework

### CODE REVIEW

File: aprotos/05HTML/form2.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+</fieldset>
+<fieldset>
+	<legend>Дата рождения</legend>
+<input type="text" name="yearbirth" required pattern="[0-9]{4}" size="4">Год рождения<br>
+<select name="month">
+	<option value="January">Январь</option>
+	<option value="February">Февраль</option>
+	<option value="March">Март</option>
+</select>Месяц<br>
+<input type="number" min=1 max=31 step="1">День
```
Comment (pluhin) [inline aprotos/05HTML/form2.html]: Where is name?

File: aprotos/05HTML/form2.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+</fieldset>
+<select name="national" required>
+<option value="RB" selected>РБ</option>
+<option value="RS">РФ</option>
+<option value="ENG">Англ</option>
+</select><br>
+Адрес: <br>
```
Comment (pluhin) [inline aprotos/05HTML/form2.html]: Label!!!

File: aprotos/05HTML/form2.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+<fieldset><legend>Данные для входа и восстановления пароля</legend>
+<input type="text" placeholder="Логин" name="login" pattern="[A-Za-z]{5,20}" required><br>
+<input type="password" placeholder="Пароль" name="password" pattern="[A-Za-z]{10,20}" required autocomplete="off">
+</fieldset>
+<fieldset><legend>Координаты</legend>
+<input type="email" name="email" placeholder="введите адрес почты"><br>
+<input type="url" name="url" placeholder="введите адрес сайта">
+</fieldset>
+<fieldset><legend>Цифры</legend>
+<input type="number" min=-2.5 max=2.5 step="0.1"><br>
```
Comment (pluhin) [inline aprotos/05HTML/form2.html]: Where is name?

File: aprotos/05HTML/tester.html [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (pluhin) [inline aprotos/05HTML/tester.html]: What is is?

**Review by pluhin — CHANGES_REQUESTED** [review]

File: aprotos/05HTML/tester.html (hunk 4 above)
Comment (kissipat) [reply aprotos/05HTML/tester.html]: used it to check myself

**Review by pluhin — APPROVED** [review]
Good, approved

**Comment by pluhin** [comment]
cold you please solve conflicts

**Comment by kissipat** [comment]
tried to di this on browser
i guess its okey now?

on my local comp i should make a merge now?

File: aprotos/05HTML/form1.html [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+			<label><input type="checkbox" name="staysystem" id="staysystem"> Оставаться в системе</label><br>
+			<input type='submit' value='Войти'><br>
+			<em>* - обязательное поле</em>
+		</fieldset>
+	</form>
+</td>
+<td>
+<form id="opros" name="opros" action="http://sc.it-academy.by/fe/form.php" method="post">
+	<fieldset><legend>Как вам?</legend>
+<label><input type="radio" name="opros" id="opros" value="nichego">ничего</label><br>
```
Comment (pluhin) [inline aprotos/05HTML/form1.html]: each id should be different

File: aprotos/05HTML/form1.html [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+</fieldset>
+</form>
+</td>
+</tr>
+<tr><th>Загрузка файла на сервер</th><th>Валидация данных HTML5</th></tr>
+<tr>
+<td>
+	<form id="file" name="file" action="http://sc.it-academy.by/fe/form.php" method="post" enctype="multipart/form-data">
+<fieldset><legend>Upload</legend>
+Файл:<br>
```
Comment (pluhin) [inline aprotos/05HTML/form1.html]: where is label?

File: aprotos/05HTML/form1.html [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+<input type="file" name="upload"><br>
+Срок хранения (года):<br>
```
Comment (pluhin) [inline aprotos/05HTML/form1.html]: where is label?

File: aprotos/05HTML/form1.html [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+<input type="text" size="5" pattern="[0-9]"> <input type="submit" value="Загрузить">
+</fieldset>
+</form>
+</td>	
+<td>
+	<form id="valid" name="valid" action="http://sc.it-academy.by/fe/form.php" method="post">
+<fieldset><legend>Строка поиска</legend>
+<input type="test" name="search" placeholder="Только русские буквы"><br>
+<em>Телефонный номер в формате</em><br>
```
Comment (pluhin) [inline aprotos/05HTML/form1.html]: where is label?

**Review by pluhin — CHANGES_REQUESTED** [review]

**Pull request merged**